added how to lint your change document (#1570)
klaytn/klaytn: HOW-TO-LINT-YOUR-CHANGE.md

# How to lint your change

This document describe how to setup automatically or manually linting your change.

## Prerequisites

> - `gofumpt` should be installed. `go install mvdan.cc/gofumpt@latest`

Reviewer: ```suggestion
- `gofumpt` should be installed. To install it, run `go install mvdan.cc/gofumpt@latest`
```

>   - go version should be equal or later than v1.18.0

Reviewer: ```suggestion
  - go version should be equal to or higher than v1.18.0
```

> - `goimports` should be installed. eg. `go install golang.org/x/tools/cmd/goimports@latest`

Reviewer: ```suggestion
- `goimports` should be installed. To install it, run `go install golang.org/x/tools/cmd/goimports@latest`
```

> ## Setup Git Hook

Reviewer: ```suggestion
## Git Hook Setup
```

> This will apply code formating automatically when you commit to git repository. So you can pass the linting tests registered in Klaytn Circle CI.

Reviewer: ```suggestion
This will apply code formatting automatically when you commit your change. So you can pass the lint tests registered in Klaytn's CI.
```

> - Copy and paste below pre-commit script to `klaytn/.git/hooks/pre-commit` file and change the file to be executable eg. `chmod +x pre-commit`

Reviewer: ```suggestion
- Copy and paste below pre-commit script to `klaytn/.git/hooks/pre-commit` file and make the file executable (e.g., `chmod +x pre-commit`).
```
